21/debian/bookworm-slim/hotspot/preview/Dockerfile

```
@@ -0,0 +1,150 @@
+ARG BOOKWORM_TAG=20230904
```

```suggestion
ARG BOOKWORM_TAG=20230919
```

To keep coherency with the `21/debian/bookworm-slim/hotspot/Dockerfile` content